doc/installing.md

```diff
@@ -24,7 +24,7 @@ sudo apt update
 sudo apt install acl attr autoconf automake dnsmasq-base git golang libacl1-dev libcap-dev liblxc1 liblxc-dev libsqlite3-dev libtool libudev-dev liblz4-dev libuv1-dev make pkg-config rsync squashfs-tools tar tcl xz-utils ebtables
 ```
 
-There are a few storage drivers for LXD besides the default directory driver.
+There are a few storage drivers for LXD besides the default `directory` driver.
```

Comment:
This one should be either "the default directory driver" (as is) or "the default `dir` driver". See https://linuxcontainers.org/lxd/docs/master/reference/storage_dir/

Reply:
I changed it to the latter.